src/software_heritage/agent/SoftwareHeritageAgent.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/*
+ Copyright (C) 2014-2019, Siemens AG
```

Comment:
There should not a copyright by Siemens here. If you want, you can add your copyright for the current year.